src/Workspaces/Core/Portable/Shared/Collections/IntervalTree`1.cs

```diff
@@ -297,11 +297,11 @@ public IEnumerator<T> GetEnumerator()
             candidates.Push((root, firstTime: true));
             while (candidates.Count != 0)
             {
-                var currentTuple = candidates.Pop();
-                var currentNode = currentTuple.node;
+                var (node, firstTime) = candidates.Pop();
```

Comment:
this seems like a similar case where the local below is redundant, your call whether to go in and simplify.